go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.16.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
 	github.com/hashicorp/terraform-svchost v0.0.0-20200729002733-f050f53b9734
-	github.com/scalr/go-scalr v0.0.0-20240701182247-6041f696e09d
+	github.com/scalr/go-scalr v0.0.0-20240704172455-7d196e31dad8
```

Comment:
@soltysss It's now pointing to an outdated commit

Reply:
updated